hw/ip/otbn/util/get_instruction_count_range.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import sys
+
+from shared.decode import decode_elf
+from shared.instruction_count_range import (program_insn_count_range,
+                                            subroutine_insn_count_range)
+
+
+def main() -> int:
```

Comment:
nit: This always returns `0`. I thought the intention for `sys.exit(main())` call was to distinguish error cases. Are you planning to update this later or does it make sense to return some other value if the maximum instruction count cannot be calculated?

Reply:
I think in this case it doesn't really make sense to return an error; the minimum instruction count is still useful on its own. The `sys.exit(main())` is just for consistency with the other OTBN util scripts, which can return 1; I could change it to `main()` instead.

Reply:
I see, thanks for the clarification. It's fine to keep this as is. Just wanted to check.

Reply:
I've changed it to `main()`, it's clearer for now and easy enough to change back :slightly_smiling_face: